### DISCUSSION TITLE
study/2023-03-08-CS-api%EB%9E%80jsonplaceholder%EC%99%80%EA%B3%B5%EA%B3%B5%EB%8D%B0%EC%9D%B4%ED%84%B0API%EC%82%AC%EC%9A%A9%EA%B8%B0/

### DISCUSSION THREAD

**Discussion by giscus** [suakang17/suakang17.github.io#2] [Comments]
# study/2023-03-08-CS-api%EB%9E%80jsonplaceholder%EC%99%80%EA%B3%B5%EA%B3%B5%EB%8D%B0%EC%9D%B4%ED%84%B0API%EC%82%AC%EC%9A%A9%EA%B8%B0/

API란? 항상 막연하게만 느껴지던 API라는 개념이 이 쯤 되니 이해가 간다!

https://suakang17.github.io/study/2023-03-08-CS-api%EB%9E%80jsonplaceholder%EC%99%80%EA%B3%B5%EA%B3%B5%EB%8D%B0%EC%9D%B4%ED%84%B0API%EC%82%AC%EC%9A%A9%EA%B8%B0/

**Comment by suakang17** [comment]
댓글 기능을 드디어 만들었습니다!
